Components/Input/input.css

```diff
@@ -0,0 +1,62 @@
+.form-group {
+  width: 100%;
+  margin-bottom: 1rem;
```

Comment:
Use global variables for properties like margin, padding, and gap to maintain consistency across the website, like 
--space-medium:1rem
--space-xsmall:0.5rem;

Reply:
Will try to implement this atlast. I am not clear right now , what values are reused

Reply:
Okay

Reply:
When I click on the image link, this opens
![image](https://user-images.githubusercontent.com/88072012/152921433-1e0d9f64-87b9-4c46-bfa9-3f22f6eb509f.png)